<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-N7UD7B5O" failures="2" tests="2" name="test1" time="0.956" errors="0" timestamp="16 Apr 2023 11:50:54 GMT">
  <testcase classname="com.qa.test.GetAPITest" name="getAPITestwithoutHeaders" time="0.783">
    <failure type="java.lang.AssertionError" message="expected [3] but found [6]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [6]
at com.qa.test.GetAPITest.getAPITestwithoutHeaders(GetAPITest.java:65)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getAPITestwithoutHeaders -->
  <testcase classname="com.qa.test.GetAPITest" name="getAPITestwithHeaders" time="0.127">
    <failure type="java.lang.AssertionError" message="expected [3] but found [6]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [6]
at com.qa.test.GetAPITest.getAPITestwithHeaders(GetAPITest.java:121)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getAPITestwithHeaders -->
</testsuite> <!-- test1 -->
